src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,301 @@ metadata_model(
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+`LinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features (identity link function).
+Options exist to specify an intercept or offset feature.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `schema(y)`
+
+# Hyper-parameters
+
+- `fit_intercept=true`: Whether to calculate the intercept for this model.
+   If set to false, no intercept will be calculated (e.g. the data is expected
+   to be centered)
+- `allowrankdeficient=false`: Whether to allow rank deficient matrices (e.g.
+   more columns than rows)
+- `offsetcol=nothing`: Name of the column to be used as an offset, if any.
+   An offset is a variable which is known to have a coefficient of 1.
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
+   features `Xnew` having the same Scitype as `X` above. Predictions are
+   probabilistic.
+- `predict_mean(mach, Xnew)`: instead return the mean of
+   each prediction above
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `features`: The names of the features used during model fitting
+- `coef`: The linear coefficients determined by the model
+- `intercept`: The intercept determined by the model
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `deviance`: Measure of deviance of fitted model with respect to
+   a perfectly fitted model. For a linear model, this is the weighted
+   residual sum of squares
+- `dof_residual`: degrees of freedom for residuals, when meaningful
+- `stderror`: standard errors of the coefficients
+- `vcov`: estimated variance-covariance matrix of the coefficient estimates
+
+# Examples
+
+```
+using MLJ
+GLM = @load LinearRegressor pkg=GLM
+glm = GLM()
+
+X, y = make_regression(100, 2) # synthetic data
+mach = machine(glm, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 2)
+yhat = predict(mach, Xnew) # new predictions
+yhat_point = predict_mean(mach, Xnew) # new predictions
+
+fitted_params(mach).features
+fitted_params(mach).coef # x1, x2, intercept
+fitted_params(mach).intercept
+
+report(mach)
+```
+
+See also
+[`LinearCountRegressor`](@ref), [`LinearBinaryClassifier`](@ref)
+"""
+LinearRegressor
+
+"""
+$(MMI.doc_header(LinearBinaryClassifier))
+
+`LinearBinaryClassifier` is a [generalized linear model](https://en.wikipedia.org/wiki/Generalized_linear_model#Variance_function), specialised
+to the case of a binary target variable, with a user-
+specified link function. Options exist to specify an intercept or offset feature.
```

Comment:
Remove some whitespace around hyphen in "user-specified":

```suggestion
to the case of a binary target variable, with a user-specified link function. 
Options exist to specify an intercept or offset feature.
```